apollo-client/src/main/java/com/ctrip/framework/apollo/util/factory/PropertiesFactory.java

```diff
@@ -0,0 +1,11 @@
+package com.ctrip.framework.apollo.util.factory;
+
+import com.ctrip.framework.apollo.util.OrderedProperties;
+import java.util.Properties;
+
+public class PropertiesFactory {
+
+  public static Properties getPropertiesObject(){
+    return new OrderedProperties();
```

Comment:
How about we adding a flag here to enable this ordered properties feature only if user intends to?

Reply:
Good idea! Implemented in latest commits.